src/main/java/nl/tudelft/contextproject/controller/GameController.java

```diff
@@ -34,6 +40,28 @@ public GameController(SimpleApplication app, Level level) {
 		super(app, "GameController");
 		game = new Game(level);
 	}
+	
+	/**
+	 * Create a game with a level loaded from a file.
+	 * @param app The main app that this controller is attached to.
+	 * @param file The file where to load the level from.
+	 */
+	public GameController(SimpleApplication app, File file) {
+		super(app, "GameController");
+		List<Entity> entities = new ArrayList<>();
+		List<Light> lights = new ArrayList<>();
+		String[] tmp = file.getName().split("_")[0].split("x");
+		int w = Integer.valueOf(tmp[0]);
+		int h = Integer.valueOf(tmp[1]);
```

Comment:
Lines 54 and 55 cause FindBugs warning for unneccesary boxing/unboxing. Use `Integer.parseInt` instead.